cloudbeat.reference.yml

```diff
@@ -41,24 +41,25 @@ cloudbeat:
           "/hostfs/var/lib/etcd",
           "/hostfs/etc/kubernetes/pki"
         ]
-    # EKS Fetchers configuration
-    eks:
-      - name: kube-api
-      - name: process
-        directory: "/hostfs"
-        processes:
-          kubelet:
-            config-file-arguments:
-              - config
-      - name: aws-ecr
-      - name: aws-elb
-      - name: file-system
-        patterns: [
-          "/hostfs/etc/kubernetes/kubelet/kubelet-config.json",
-          "/hostfs/var/lib/kubelet/kubeconfig",
-        ]
-  evaluator:
-    decision_logs: false
+
+    #    # EKS Fetchers configuration
+    #      - name: kube-api
+    #      - name: process
+    #        directory: "/hostfs"
+    #        processes:
+    #          kubelet:
+    #            config-file-arguments:
+    #              - config
+    #      - name: aws-ecr
+    #      - name: aws-elb
+    #      - name: file-system
+    #        patterns: [
+    #          "/hostfs/etc/kubernetes/kubelet/kubelet-config.json",
+    #          "/hostfs/var/lib/kubelet/kubeconfig",
+    #        ]
```

Comment:
nit: indentation is off for this part, also in other files

Reply:
fixed